pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/TenantsBase.java

```diff
@@ -105,6 +105,13 @@ public void createTenant(
 
         try {
             NamedEntity.checkName(tenant);
+            List<String> tenants = globalZk().getChildren(path(POLICIES), false);
```

Comment:
You shouldn't get the list of tenants from zookeeper first. You should check `maxTenants` first. If `maxTenants` is larger than 0, this feature is enabled. Then you get the list of tenants.